pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -176,7 +178,8 @@ func PostStartHook(hookContext genericapiserver.PostStartHookContext) error {
 		}
 
 		// ensure bootstrap rolebindings are created or reconciled
-		for _, clusterRoleBinding := range append(bootstrappolicy.ClusterRoleBindings(), bootstrappolicy.ControllerRoleBindings()...) {
+		bindNodeRole := !utilfeature.DefaultFeatureGate.Enabled(features.NodeAuthorizer)
+		for _, clusterRoleBinding := range append(bootstrappolicy.ClusterRoleBindings(bindNodeRole), bootstrappolicy.ControllerRoleBindings()...) {
```

Comment:
I'd like to do the following:
* make enabling the node authorizer skip this binding (implemented in this PR)
* announce the default `system:nodes` binding as deprecated now, and remove it completely in 1.8/1.9 timeframe, at which point we can get rid of this flag.